packages/ui/src/Popup/ProvideUidRequests/Request.tsx

```diff
@@ -150,8 +150,8 @@ function Request ({ isFirst, reqId, request, url }: Props): React.ReactElement<P
               </Flex>
               <Text color='gray.1'
                 variant='b2m'>
-                You already have an existing uID assigned to the current Polymesh identity, accepting this uID will
-                overwrite the exisiting one. Do you like to accept?
+                You already have an existing uID assigned to the current Polymesh Account, accepting this uID will
+                overwrite the existing one. Do you like to accept?
```

Comment:
This doesn't sound right to me. I think it should be "Do you accept?" or "Would you like to accept?" Or even "Do you want to proceed?"

Reply:
Good catch! Definitely this needs to be changed to "Would you like to accept?". Thanks!